issue_template.md

```diff
@@ -1,29 +1,41 @@
-### Summary
-
--   What does this package do? (explain in 50 words or less):
+Submitting Author: Name (@github_handle)  
+Repository:   
+Version submitted:   
+Editor: TBD  
+Reviewer 1: TBD  
+Reviewer 2: TBD  
+Archive: TBD  
+Version accepted: TBD   
 
+---
 
 -   Paste the full DESCRIPTION file inside a code block below:
 
 ```
 
 ```
 
--   URL for the package (the development repository, not a stylized html page):
 
-- Please indicate which category or categories from our [package fit policies](https://github.com/ropensci/onboarding/blob/master/policies.md#package-fit) this package falls under ***and why**(? (e.g., data retrieval, reproducibility. If you are unsure, we suggest you make a pre-submission inquiry.):
+## Scope 
+
+- Please indicate which category or categories from our [package fit policies](https://ropensci.github.io/dev_guide/policies.html#package-categories) this package falls under ***and why**(? (Please check an appropriate box below. If you are unsure, we suggest you make a pre-submission inquiry.):
 
-[e.g., "data extraction, because the package parses a scientific data file format"]
+	- [ ] data retrieval
+	- [ ] data extraction
+	- [ ] database access
+	- [ ] data munging
+	- [ ] data deposition
+	- [ ] reproducibility
+	- [ ] geospatial data
+	- [ ] text analysis
 
 
--   Who is the target audience and what are scientific applications of this package?  
+-   Who is the target audience and what are scientific applications of this package?  
 
--   Are there other R packages that accomplish the same thing? If so, how does
-yours differ or meet [our criteria for best-in-category](https://github.com/ropensci/onboarding/blob/master/policies.md#overlap)?
+-   Are there other R packages that accomplish the same thing? If so, how does yours differ or meet [our criteria for best-in-category](https://github.com/ropensci/onboarding/blob/master/policies.md#overlap)?
 
--   If you made a pre-submission enquiry, please paste the link to the corresponding issue, forum post, or other discussion, or @tag the editor you contacted.
+-   If you made a pre-submission enquiry, please paste the link to the corresponding issue, forum post, or other discussion, or @tag the editor you contacted.
 
-### Requirements
 
 Confirm each of the following by checking the box.  This package:
```

Comment:
it looks like this section is under the __Scope__ heading

Reply:
Good point. I meant to add technical checks or something like that. Do you have a suggestion for a sub-heading?

Reply:
technical checks sounds good